pkgs/by-name/gp/gpick/package.nix

```diff
@@ -28,7 +27,12 @@ stdenv.mkDerivation rec {
     ./dot-version.patch
 
     (fetchpatch {
-      url = "https://raw.githubusercontent.com/archlinux/svntogit-community/1d53a9aace4bb60300e52458bb1577d248cb87cd/trunk/buildfix.diff";
+      url = "https://patch-diff.githubusercontent.com/raw/thezbyg/gpick/pull/227.patch";
+      hash = "sha256-qYspUctvlPMEK/c2hMUxYc5EYdG//CBcN2PluTtXiFc=";
+    })
+
+    (fetchpatch {
+      url = "https://gitlab.archlinux.org/archlinux/packaging/packages/gpick/-/raw/0.3-2/buildfix.diff";
       hash = "sha256-DnRU90VPyFhLYTk4GPJoiVYadJgtYgjMS4MLgmpYLP0=";
```

Comment:
Perhaps this no-op url move should be done in a different commit?

Reply:
checked that the hash is unchanged, lgtm